backend/tests/unit/simulation/alarms.txt

```diff
@@ -0,0 +1,34 @@
+Scenario: alarm services behaving properly
+Given: a pc_ac alarm service object 
+When: parameter mode is not equal to pc_ac
+Then: alarms are not updated for parameters
+
+Scenario: alarm services behaving properly
+Given: a pc_ac alarm service object 
+When: parameter ventilating mode is false
+Then: alarms are not updated for parameters
+
```

Comment:
Presumably in a future (lower-priority) PR we will also test what happens when the value is within the limits: if no alarm is active on that value, nothing changes; if an alarm was previously active on that value, then the alarm gets deactivated.